conf/default.toml

```diff
@@ -6,7 +6,7 @@
 [global]
 # This URL (relative or absolute) must point to the API endpoint of a GeoNetwork4 instance
 geonetwork4_api_url = "/geonetwork/srv/api"
-datahub_url = "/datahub"
+datahub_url = "http://myhost/datahub"
```

Comment:
this doesn't look like a good default for everyone; why not keep `/datahub`? btw one of the datahub e2e tests uses a custom configuration file

Reply:
This method crashes if url is not  https://github.com/geonetwork/geonetwork-ui/blob/a12b13c742694b340efea9d196d625fc2a505bb9/apps/metadata-editor/src/app/router.service.ts#L21

Reply:
Right, this is a common problem. That's because the URL constructor will throw if it's provided with a relative url and no second argument; in that case the typical solution is to use:
```
new URL(`${getGlobalConfig().DATAHUB_URL}/search`, window.location.toString())
```
This will make a relative URL use the current host.